design-proposals/vpro-device.md

```diff
@@ -85,11 +85,14 @@ sequenceDiagram
     deactivate en
 
     en ->> en: 5. OS installation (includes Agent RPMs)
-    en ->> agent: 6. Install/Enable Agent as part of OS
+    
+    en ->> en: 5a. Installer check hardware capability for vPRO/ISM support
```

Comment:
Good to callout similar logic is used for EMT in the cloudinit to disable the PMA if device is not vpro/ISM.